components/engine_rocks/src/cf_options.rs

```diff
@@ -40,6 +40,23 @@ impl RocksCfOptions {
     pub fn into_raw(self) -> RawCfOptions {
         self.0
     }
+
+    pub fn set_flush_size(&mut self, f: usize) -> Result<()> {
+        if let Some(m) = self.0.get_write_buffer_manager() {
+            m.set_flush_size(f);
+        } else {
+            return Err(box_err!("write buffer manager not found"));
```

Comment:
Do you need to know the cf name?

Reply:
I think is unnecessary, as the client should know the cf. And seems we cannot fetch the cf name here.